config/config.go

```diff
@@ -941,8 +941,12 @@ func (cfg *FastSyncConfig) ValidateBasic() error {
 // including timeouts and details about the WAL and the block structure.
 type ConsensusConfig struct {
 	RootDir string `mapstructure:"home"`
-	WalPath string `mapstructure:"wal_file"`
-	walFile string // overrides WalPath if set
+	// Experimental: If set to true, only internal messages will be written
+	// to the WAL. External messages like votes, proposals
+	// block parts, will not be written
+	OnlyInternalWal bool   `mapstructure:"only_internal_wal"`
```

Comment:
nit:

we might want to be more explicit with the default value being false

Reply:
done